Working-With-Engineers.md

```diff
@@ -0,0 +1,42 @@
+# Working With CoverHound Engineers:
+## What to know about our working agreements
+
+### [Core Hours:](./people/Communication.md) 10:30AM – 3:30PM
+
+Engineers are expected to be available for communication through Slack, phone,
+or in person, and able to respond promptly (barring meetings) during these
+corehours. It is up to each individual engineer to determine what hours outside
+of those will make up the rest of their day.
+
+### [Slack and Day-to-Day Communication](./people/Communication.md)
+
+Please err on the side of messaging someone in Slack before approaching them in
+person. Engineering often requires deep thinking; interruptions and switching
+context can be a big setback.
+
+### [Meetings](./people/Meetings.html)
```

Comment:
Meetings.md?

Reply:
Whoops, thanks for catching! Pushed up a change.